src/curated-corpus/components/ScheduleItemFormConnector/ScheduleItemFormConnector.test.tsx

```diff
@@ -172,13 +172,14 @@ describe('ScheduleItemFormConnector', () => {
     userEvent.click(datePicker);
 
     // Find today's date on the monthly calendar (it's actually a button)
-    const date = screen.getByRole('button', {
+    const dates = screen.getAllByRole('button', {
       name: today.toFormat('d'),
-    }) as HTMLButtonElement;
+    }) as HTMLButtonElement[];
```

Comment:
I don't think we need to cast it to `HTMLButtonElement` if we fetch them by the button role 🤔

Reply:
It's often needed, but after whatever happened in `main` that broke this test it's no longer required - I'll remove it 👍.